angular-portfolio/src/app/awards.service.ts

```diff
@@ -1,14 +1,21 @@
 import { Injectable } from '@angular/core';
 import { Awards } from './awards-template';
-import { AWARD } from './mock-awards';
 import { Observable, of } from 'rxjs';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
 
 @Injectable({
   providedIn: 'root'
 })
+
 export class AwardsService {
 
+  private award = '/load-awards';
```

Comment:
A lot of things are named award in this pr, and it can get confusing. This in particular should be something like "listAwardsResource".